malaria-ios/ViewControllers/UserProfilePillTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import DoneToolbarSwift
+
+// MARK: - Table View Cell
+
+class UserProfilePillTableViewCell: UITableViewCell {
+  
+  @IBOutlet weak var pillName: UILabel!
+  @IBOutlet weak var quantityTextField: UITextField!
+  
+  weak var delegate : SaveRemainingPillsProtocol?
+  
+  var toolBar: ToolbarWithDone!
+  
+  func updateCellWithParameters(delegate: SaveRemainingPillsProtocol,
```

Comment:
"WithParamenters" is redundant.

Weird indentation, is your screen lacking width? Personally, I would put everything on the same line since it improves redability and this method doesn't have so many parameters.